Description: Use automake's "foreign" option to fix FTBFS during autoreconf.

--- a/configure.ac
+++ b/configure.ac
@@ -33,7 +33,7 @@ AC_CONFIG_SRCDIR([pbs_drmaa/drmaa.c])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_AUX_DIR([scripts])
 
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign])
 
 # command-line arguments:
 AC_ARG_ENABLE(debug, AC_HELP_STRING([--enable-debug],
--- pbs-drmaa-1.0.17.orig/drmaa_utils/configure.ac
+++ pbs-drmaa-1.0.17/drmaa_utils/configure.ac
@@ -32,7 +32,7 @@ AC_CONFIG_SRCDIR([drmaa_utils/xmalloc.c]
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_AUX_DIR([scripts])
 
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign])
 DEVELOPER_MODE="no"
 AM_CONDITIONAL([DEVELOPER_MODE], [test "x$DEVELOPER_MODE" = "xyes"])
 
